htdocs/public/onlinesign/newonlinesign.php

```diff
@@ -452,13 +452,34 @@
 			print '<br>';
 			if ($message == 'signed') {
 				print '<span class="ok">'.$langs->trans("PropalSigned").'</span>';
+				//fix #21941 run trigger
+				if (method_exists($object, 'call_trigger')) {
```

Comment:
Why calling the method into the public page and not into the method that set the status so into 
core/ajax/onlineSign.php

Reply:
why .... hmmm because i have to make a choice 🔀  but it seems to be the bad one i've made, sorry 👯

Reply:
so ? would you prefer that i switch that code from newonlinesign.php to ajax/onlineSign.php ?

Reply:
Yes trigger must be nearest the code that justify its execution.

Reply:
yes sir ! i will do it now

Reply:
ok, i've moved one trigger to ajax/onlineSign.php but for the other (refused) there is no other place than newonlinesign.php
... sorry :)

commit & PR update will become in some sec / min depends on my git-power !

Reply:
PR is clean and ready :)